azavea/temperate
# Show indicator charts in Assessment Wizard and risk summary popover

## Overview
Adds a modal to show indicator charts for the Assessment Wizard and risk popover. Uses live data, but fixed for Philadelphia - see #404.

Also adds a generic modal template component that we can use for consistent functionality and styling. And it even closes too!

### Demo
Risk popover modal:
![screen shot 2018-01-08 at 3 04 22 pm](https://user-images.githubusercontent.com/1032849/34689939-4477ac7a-f485-11e7-8620-ae57cbcb87f3.png)

"View Related Indicators" button and expanded page call-to-action:
<img width="817" alt="screen shot 2018-01-05 at 3 01 01 pm" src="https://user-images.githubusercontent.com/1032849/34626082-4938511e-f229-11e7-96ef-84c7695c7268.png">

"View Related Indicators" modal, all collapsed:
![screen shot 2018-01-08 at 2 14 07 pm](https://user-images.githubusercontent.com/1032849/34687916-441e3b74-f47e-11e7-9e02-4043f88ae35e.png)

"View Related Indicators" modal, one expanded:
![screen shot 2018-01-08 at 2 14 47 pm](https://user-images.githubusercontent.com/1032849/34687927-4ff1f5e4-f47e-11e7-93c5-4d5b2e58671c.png)

## Testing Steps
- In the "Vulnerability Assessment" page, click on a risk name to open the popover, then click on an associated indicator
  - A modal should appear showing the indicator's chart for Philadelphia (See #404)
- Click "Assess" on a non-assessed risk, progress to step 2, "Hazard", and click "View related indicators"
  - You should see a list of indicators in blinds, like the Indicators page 
  - Opening a blind should show that indicator's chart for Philadelphia

Closes #238
